['Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpj3ol7vsd/solution_0.py", line 66, in <module>\n    print(Solution().longestSpecialPath(edges = [[0,1,10],[0,2,15],[1,3,20],[1,4,25],[2,5,30]], nums = [1,2,3,4,5,6]))\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpj3ol7vsd/solution_0.py", line 25, in longestSpecialPath\n    adjacency[edge.u].append((edge.v, edge.w))\nAttributeError: \'list\' object has no attribute \'u\'\n', 'Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpj3ol7vsd/solution_1.py", line 66, in <module>\n    print(Solution().longestSpecialPath(edges = [[0,1,1],[1,2,1],[2,3,1],[3,4,1],[4,5,1]], nums = [1,2,3,4,5,6]))\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpj3ol7vsd/solution_1.py", line 25, in longestSpecialPath\n    adjacency[edge.u].append((edge.v, edge.w))\nAttributeError: \'list\' object has no attribute \'u\'\n']